CHANGELOG_iOS.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.67.127](https://github.com/brave/brave-browser/releases/tag/v1.67.127)
+
+### Web3
+
+- Added support for retrying transactions on the Solana network. ([#37443](https://github.com/brave/brave-browser/issues/37443))
+- Added fee waiver for swaps in Brave Wallet. ([#38566](https://github.com/brave/brave-browser/issues/38566))
```

Comment:
Change to `Removed “Brave Fee” for all “Swap” transactions in Brave Wallet.` to keep it same for all platform